apps/demo/app/custom-ui/[...puckPath]/client.tsx

```diff
@@ -361,6 +362,13 @@ export function Client({ path, isEdit }: { path: string; isEdit: boolean }) {
           duplicate: false,
           insert: false,
         },
+        resolvePermissions: (data) => {
+          if (data.props.title.includes("Puck")) {
+            return { duplicate: true };
+          } else {
+            return { duplicate: false };
+          }
+        },
```

Comment:
Not a blocking comment, but I'll likely tweak these examples before release as I'd like them in the main demo config with a more real-world use-case